cpp/src/ray/runtime/task/native_task_submitter.cc

```diff
@@ -83,5 +86,28 @@ ObjectID NativeTaskSubmitter::SubmitActorTask(InvocationSpec &invocation,
   return Submit(invocation, task_options);
 }
 
+std::string NativeTaskSubmitter::GetFullName(bool global, const std::string &name) {
+  if (name.empty()) {
+    return "";
+  }
+  return global ? name
+                : CoreWorkerProcess::GetCoreWorker().GetCurrentJobId().Hex() + "-" + name;
+}
+
+ActorID NativeTaskSubmitter::GetActor(bool global, const std::string &actor_name) {
+  auto &core_worker = CoreWorkerProcess::GetCoreWorker();
+  auto full_actor_name = GetFullName(global, actor_name);
+  auto pair = core_worker.GetNamedActorHandle(full_actor_name, "");
+  if (!pair.second.ok()) {
+    RAY_LOG(WARNING) << pair.second.message();
+    return ActorID::Nil();
+  }
+
+  std::string actor_id;
+  auto actor_handle = pair.first;
+  RAY_CHECK(actor_handle);
+  return actor_handle->GetActorID();
+}
```

Comment:
Same here

Reply:
GetActor is a virtual function, one is for native mode, another is for local mode.